src/data/faq_de.json

```diff
@@ -138,9 +138,9 @@
-                            "<div class='container'><div class='row'><div class='col-sm'><a href='https://map.schnelltestportal.de' target='_blank' rel='noopener'><img class='img-fluid' src='../../assets/img/faq/schnelltestportal_1.jpeg' alt='image'></a></div><div class='col-sm'>Mittlerweile wurden mehr als 100 Schnelltest-Partner an die CWA-Infrastruktur angeschlossen und können die Ergebnisse in die App übermitteln.<br>Eine lokale Suche, die Ihnen in der Umgebung die kompatiblen Partner mit den Teststellen anzeigt, finden Sie hier: <a href='https://map.schnelltestportal.de' target='_blank' rel='noopener noreferrer'>map.schnelltestportal.de</a><br>Die Liste der aufgeführten Teststellen enthält nur die Einträge, die von den betreffenden Partnern gemeldet wurden.</div></div></div>",
+                            "<div class='container'><div class='row'><div class='col-sm'><a href='https://map.schnelltestportal.de' target='_blank' rel='noopener'><img class='img-fluid' src='../../assets/img/faq/schnelltestportal_1.jpeg' alt=''></a></div><div class='col-sm'>Mittlerweile wurden mehr als 100 Schnelltest-Partner an die CWA-Infrastruktur angeschlossen und können die Ergebnisse in die App übermitteln.<br>Eine lokale Suche, die Ihnen in der Umgebung die kompatiblen Partner mit den Teststellen anzeigt, finden Sie hier: <a href='https://map.schnelltestportal.de' target='_blank' rel='noopener noreferrer'>map.schnelltestportal.de</a><br>Die Liste der aufgeführten Teststellen enthält nur die Einträge, die von den betreffenden Partnern gemeldet wurden.</div></div></div>",
                             "<hr/>",
-                            "<div class='container'><div class='row'><div class='col-sm'><a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener'><img class='img-fluid' src='../../assets/img/faq/BMG_testzentren.png' alt='image'></a></div><div class='col-sm'>Eine Übersicht der <b>PCR- und Schnell-Teststellen</b> finden Sie auf dieser <a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener noreferrer'>BMG Seite</a> (nicht ersichtlich, ob die Teststelle an die CWA angebunden ist oder nicht).</div></div>",
+                            "<div class='container'><div class='row'><div class='col-sm'><a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener'><img class='img-fluid' src='../../assets/img/faq/BMG_testzentren.png' alt=''></a></div><div class='col-sm'>Eine Übersicht der <b>PCR- und Schnell-Teststellen</b> finden Sie auf dieser <a href='https://www.zusammengegencorona.de/testen/?articlefilter=alleartikel' target='_blank' rel='noopener noreferrer'>BMG Seite</a> (nicht ersichtlich, ob die Teststelle an die CWA angebunden ist oder nicht).</div></div>",

```

Comment:
Not sure about this one. This seems to be an empty link because the link has no text, only an image. It would make sense to add something useful here.